certbot/certbot/_internal/cli/__init__.py

```diff
@@ -4,7 +4,11 @@
 import logging
 import logging.handlers
 import sys
+from typing import Any
+from typing import Iterable
```

Comment:
I have to admit that it is quite puzzling.

For `Any` itself, it is used directly by several functions like `has_default_value` and `option_was_set`, but for `Iterable` it is not the case indeed, and it has certainly been added by me on a previous stage of this PR or its aggregated parent.

For why pylint is not complaining, I found a similar issue for pyflakes that gives more strength to my assumption. Basically since `__init__.py` is a little special since it offers the opportunity to "expose" an importable component of a package for alternative situations where the importable component is a module of the package (the usual situation). For this specific purpose I think `pylint`, as `pyflakes`, is not complaining about the unused imports since they may be used on third parties.

This is basically how `certbot._internal.cli.__init__` has been designed since its introduction to split the `certbot._internal.cli` module. Now it is less interesting given that the module is internal, and several other imports may also be removed as well. I think it could be done in another PR.

Reply:
To address the primary issue, I remove `from typing import Iterable` since it is an import type unused and introduced by this PR.